docs/docs/guides/web3_plugin_guide/plugin_authors.md

```diff
@@ -240,148 +231,70 @@ export class CustomRpcMethodsPlugin extends Web3PluginBase {
 }
 
 // Module Augmentation
-declare module './reexported_web3_context' {
-	interface Web3Context {
-		customRpcMethods: CustomRpcMethodsPlugin;
-	}
-}
-
-export { Web3Context };
-```
-
-The following represent what your plugin users would see:
-
-![web3 context augmentation](./assets/web3_context_augmentation.png 'web3Context augmentation')
-
-The above screenshot shows IntelliSense thinking `.customRpcMethods.someMethod` is available to call on the instance of `Web3`, even though the plugin user hasn't registered `CustomRpcMethodsPlugin` - running this code would result in an error.
```

Comment:
I would consider adding this back in as it demonstrates visually the negative result of module augmentation